src/util/tokens.ts

```diff
@@ -5,6 +5,11 @@ export const tokenAmountInUnits = (amount: BigNumber, decimals: number, toFixedD
     return amount.div(decimalsPerToken).toFixed(toFixedDecimals);
 };
 
+export const tokenAmountInUnitsToBigNumber = (amount: BigNumber, decimals: number): BigNumber => {
```

Comment:
Req: `tokenAmountInUnits` could use this function and add the `toFixed` to the result.

Reply:
Done!